romana-install/roles/romana/install-master/templates/config/romana.conf.yml

```diff
@@ -63,6 +63,9 @@ services:
     config:
       lease_file : "/etc/ethers"
       wait_for_iface_try: 6
+      store:
+        type: sqlite3
+        database: /var/agent.db
```

Comment:
The reason why we care about storage for agent in first place is to ensure that data (aka desired state) will be preserved through agent restarts and host restarts.